runtime/src/main/kotlin/kotlin/native/concurrent/Atomics.kt

```diff
@@ -216,8 +216,7 @@ private fun debugString(value: Any?): String {
 /**
  * An atomic reference to a frozen Kotlin object. Can be used in concurrent scenarious
  * but frequently shall be of nullable type and be zeroed out once no longer needed.
- * Otherwise memory leak could happen. To detect such leaks [kotlin.native.internal.GC.detectCycles]
- * in debug mode could be helpful.
+ * Asynchronous cycle collector takes care of cyclic references of that kind.
```

Comment:
Is "shall be of nullable type and be zeroed out once no longer needed" above left intentionally?

Reply:
Yes, it kind of suggest people to think about cleaning up resources.